reference/orders.v3.yml

```diff
@@ -2043,9 +2043,51 @@ components:
                   type: number
                   format: float
                   example: 35.42
-      title: Transaction
-      x-internal: false
       type: object
+      title: ''
+      x-examples:
+        Example 1:
+          event: purchase
+          method: credit_card
+          amount: -3.402823669209385e+38
```

Comment:
Thinking that this here needs to be adjusted.

Reply:
Good find @bc-tgomez. I made the number positive.